app/views/merchants/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Merchants</h1>
+
+<% @merchants.each do |merchant| %>
+      <p><%= link_to "#{merchant.name}", "/merchants/#{merchant.id}" , params: {:merchant_id => merchant.id} %> </p>
```

Comment:
very minor comment but I find the newer syntax for params more readable:
`params: {merchant_id: merchant.id}`